### ISSUE TITLE
Unable to install - Unable to load configuration flow: undefined

### ISSUE THREAD

**Issue by fraintt** [music-assistant/hass-music-assistant#497]
### What version of Music Assistant has the issue?

2022.6.4

### The problem

I´m unable to install MusicAssistant, Im adding it via HACS, and after I restarted and I went to add the Integration the error "Unable to load configuration flow: undefined" is being shown. (Not sure if translated correctly, as it displays in my native language). I also tried to remove it from HACS, restart, re-install from HACS but the same result. I was not able to find any issue even simillar to mine here on Github. 

Thanks for any help.

### How to reproduce

Added Music Assistant via HACS
Restarted HASS
Tried to add Integration via the Integrations Page

### Relevant log output

Logger: homeassistant.util.package
Source: util/package.py:99
First occurred: 22:03:11 (3 occurrences)
Last logged: 22:05:39

Unable to install package music-assistant==1.4.5: error: subprocess-exited-with-error × python setup.py bdist_wheel did not run successfully. │ exit code: 1 ╰─> [76 lines of output] running bdist_wheel running build running build_py creating build creating build/lib.linux-aarch64-cpython-39 creating build/lib.linux-aarch64-cpython-39/greenlet copying src/greenlet/__init__.py -> build/lib.linux-aarch64-cpython-39/greenlet creating build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_weakref.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_version.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_tracing.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_throw.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_stack_saved.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_leaks.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_greenlet.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_generator_nested.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_generator.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_gc.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_extension_interface.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_cpp.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_contextvars.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/__init__.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests running egg_info writing src/greenlet.egg-info/PKG-INFO writing dependency_links to src/greenlet.egg-info/dependency_links.txt writing requirements to src/greenlet.egg-info/requires.txt writing top-level names to src/greenlet.egg-info/top_level.txt reading manifest file 'src/greenlet.egg-info/SOURCES.txt' reading manifest template 'MANIFEST.in' no previously-included directories found matching 'docs/_build' warning: no files found matching '*.py' under directory 'appveyor' warning: no previously-included files matching '*.pyc' found anywhere in distribution warning: no previously-included files matching '*.pyd' found anywhere in distribution warning: no previously-included files matching '*.so' found anywhere in distribution warning: no previously-included files matching '.coverage' found anywhere in distribution adding license file 'LICENSE' adding license file 'LICENSE.PSF' adding license file 'AUTHORS' writing manifest file 'src/greenlet.egg-info/SOURCES.txt' copying src/greenlet/greenlet.c -> build/lib.linux-aarch64-cpython-39/greenlet copying src/greenlet/greenlet.h -> build/lib.linux-aarch64-cpython-39/greenlet copying src/greenlet/slp_platformselect.h -> build/lib.linux-aarch64-cpython-39/greenlet creating build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/setup_switch_x64_masm.cmd -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_aarch64_gcc.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_alpha_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_amd64_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_arm32_gcc.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_arm32_ios.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_csky_gcc.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_m68k_gcc.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_mips_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_ppc64_aix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_ppc64_linux.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_ppc_aix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_ppc_linux.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_ppc_macosx.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_ppc_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_riscv_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_s390_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_sparc_sun_gcc.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_x32_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_x64_masm.asm -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_x64_masm.obj -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_x64_msvc.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_x86_msvc.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_x86_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/tests/_test_extension.c -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/_test_extension_cpp.cpp -> build/lib.linux-aarch64-cpython-39/greenlet/tests running build_ext building 'greenlet._greenlet' extension creating build/temp.linux-aarch64-cpython-39 creating build/temp.linux-aarch64-cpython-39/src creating build/temp.linux-aarch64-cpython-39/src/greenlet gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fno-semantic-interposition -fno-builtin-malloc -fno-builtin-calloc -fno-builtin-realloc -fno-builtin-free -DTHREAD_STACK_SIZE=0x100000 -fPIC -I/usr/local/include/python3.9 -c src/greenlet/greenlet.c -o build/temp.linux-aarch64-cpython-39/src/greenlet/greenlet.o error: command 'gcc' failed: No such file or directory [end of output] note: This error originates from a subprocess, and is likely not a problem with pip. ERROR: Failed building wheel for greenlet error: subprocess-exited-with-error × Running setup.py install for greenlet did not run successfully. │ exit code: 1 ╰─> [78 lines of output] running install /usr/local/lib/python3.9/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools. warnings.warn( running build running build_py creating build creating build/lib.linux-aarch64-cpython-39 creating build/lib.linux-aarch64-cpython-39/greenlet copying src/greenlet/__init__.py -> build/lib.linux-aarch64-cpython-39/greenlet creating build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_weakref.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_version.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_tracing.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_throw.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_stack_saved.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_leaks.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_greenlet.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_generator_nested.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_generator.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_gc.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_extension_interface.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_cpp.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/test_contextvars.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/__init__.py -> build/lib.linux-aarch64-cpython-39/greenlet/tests running egg_info writing src/greenlet.egg-info/PKG-INFO writing dependency_links to src/greenlet.egg-info/dependency_links.txt writing requirements to src/greenlet.egg-info/requires.txt writing top-level names to src/greenlet.egg-info/top_level.txt reading manifest file 'src/greenlet.egg-info/SOURCES.txt' reading manifest template 'MANIFEST.in' no previously-included directories found matching 'docs/_build' warning: no files found matching '*.py' under directory 'appveyor' warning: no previously-included files matching '*.pyc' found anywhere in distribution warning: no previously-included files matching '*.pyd' found anywhere in distribution warning: no previously-included files matching '*.so' found anywhere in distribution warning: no previously-included files matching '.coverage' found anywhere in distribution adding license file 'LICENSE' adding license file 'LICENSE.PSF' adding license file 'AUTHORS' writing manifest file 'src/greenlet.egg-info/SOURCES.txt' copying src/greenlet/greenlet.c -> build/lib.linux-aarch64-cpython-39/greenlet copying src/greenlet/greenlet.h -> build/lib.linux-aarch64-cpython-39/greenlet copying src/greenlet/slp_platformselect.h -> build/lib.linux-aarch64-cpython-39/greenlet creating build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/setup_switch_x64_masm.cmd -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_aarch64_gcc.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_alpha_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_amd64_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_arm32_gcc.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_arm32_ios.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_csky_gcc.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_m68k_gcc.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_mips_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_ppc64_aix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_ppc64_linux.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_ppc_aix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_ppc_linux.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_ppc_macosx.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_ppc_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_riscv_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_s390_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_sparc_sun_gcc.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_x32_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_x64_masm.asm -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_x64_masm.obj -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_x64_msvc.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_x86_msvc.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/platform/switch_x86_unix.h -> build/lib.linux-aarch64-cpython-39/greenlet/platform copying src/greenlet/tests/_test_extension.c -> build/lib.linux-aarch64-cpython-39/greenlet/tests copying src/greenlet/tests/_test_extension_cpp.cpp -> build/lib.linux-aarch64-cpython-39/greenlet/tests running build_ext building 'greenlet._greenlet' extension creating build/temp.linux-aarch64-cpython-39 creating build/temp.linux-aarch64-cpython-39/src creating build/temp.linux-aarch64-cpython-39/src/greenlet gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fno-semantic-interposition -fno-builtin-malloc -fno-builtin-calloc -fno-builtin-realloc -fno-builtin-free -DTHREAD_STACK_SIZE=0x100000 -fPIC -I/usr/local/include/python3.9 -c src/greenlet/greenlet.c -o build/temp.linux-aarch64-cpython-39/src/greenlet/greenlet.o error: command 'gcc' failed: No such file or directory [end of output] note: This error originates from a subprocess, and is likely not a problem with pip. error: legacy-install-failure × Encountered error while trying to install package. ╰─> greenlet note: This is an issue with the package mentioned above, not pip. hint: See above for output from the failure. WARNING: You are using pip version 22.0.3; however, version 22.1.2 is available. You should consider upgrading via the '/usr/local/bin/python3 -m pip install --upgrade pip' command.

### Additional information

I have spotcast and spotify integration added. 

Version | core-2022.6.4
Installation Type | Home Assistant Supervised
Development | false
Supervisor | true
Docker | true
User | root
Virtual Environment | false
Python Version | 3.9.12
Operating System Family | Linux
Operating System Version | 5.10.60-v8+
CPU Architecture | aarch64
Timezone | Europe/Warsaw

Host Operating System | Raspbian GNU/Linux 10 (buster)
Update Channel | stable
Supervisor Version | supervisor-2022.05.3
Agent Version |  
Docker Version | 20.10.8

### What version of Home Assistant Core are your running

2022.6.4

### What type of installation are you running?

Home Assistant Supervised

### On what type of hardware are you running?

Raspberry Pi

**Comment by erkr** [comment]
> I have spotcast and spotify integration added.

Those co-exist just fine.

Most likely this is related to your Supervised install. Wait for Marcel to look into your error.

**Comment by marcelveldt** [comment]
You are running an unsupported installation of Home Assistant, I think you will see that in big red letters on the device info page. This error means that a dependency (greenlet) can not be found nor built for your installation. You will have to compile that yourself if you want to make it work.

I can't help you with that, sorry.

I see you run a raspberry pi. I'd recommend to just install HAOS on it and save yourself a whole lot of trouble.

**Comment by sgueniot** [comment]
Hi, on my raspberry with HA core in docker I have a similar error. The problem (visible int the log : ` command 'gcc' failed: No such file or directory`) was that I needed to install gcc and some other dependencies in the docker container.
I am using portainer, I just opened the HA container console, it is based on alpine linux so I started the installation with 

```
apk update
apk add build-base
```
After the reboot, I restart the Music Assistant installation successfully.
I hope this helps someone
